components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
@elliott-davis does the channel need to be included in the census entry message? Would it suffice to create a new event server message that captures the channel of the service group? That would allow us to leave the current census entry as is.